desktop/release-notes.md

```diff
@@ -23,6 +23,95 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+        </div>
+      </div>
+    </div>
+  </div>
+
+### New
+
+- Added two new extensions (vcluster and PGAdmin4) are available in the Extensions Marketplace.
+- Ability to sort extensions has been added to the Extensions Marketplace.
+- No more feedback nagging - thanks to all your feedback we were able to identify a bug with our feedback dialog, in 4.11+ you should only experience the NPS twice a year.
+- Custom theme settings for Docker Desktop allow you to specify dark or light mode for Docker Desktop independent of your device settings.
+
+### Upgrades
+- [Docker Compose v2.7.0](https://github.com/docker/compose/releases/tag/v2.7.0)
+- [Docker Compose "Cloud Integrations" v1.0.28](https://github.com/docker/compose-cli/releases/tag/v1.0.28)
+- [Kubernetes v1.24.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.2)
+- [Go 1.18.4](https://github.com/golang/go/releases/tag/go1.18.4)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Add new flag for Windows installer, `--no-windows-containers` to disable Windows Containers integration.
+- Adds the Container / Compose icon as well as the exposed port(s) / exit code to the Containers screen.
+- Updated Docker theme palette colours values to match design system.
+- Improved an error message from `docker login` if Registry Access Management is blocking the Docker engine's access to Docker Hub.
+- Increase throughput between the Host and Docker, for example increasing performance of "docker cp".
+- Collecting diagnostics takes less time to complete.
+- Selecting/unselecting a compose app on the containers overview will select/unselect all its containers.
+- Show tag names on the container overview image column.
+- Added search decorations to the terminal's scrollbar so that matches outside the viewport are visible.
+- Fixed issue on search which doesn't work well on containers page [docker/for-win#12828](https://github.com/docker/for-win/issues/12828).
+- Fixed issue on infinite loading on volume screen [docker/for-win#12789](https://github.com/docker/for-win/issues/12789).
+- Fixed a problem in the Container UI where resizing or hiding columns didn't work. Fixed [docker/for-mac#6391](https://github.com/docker/for-mac/issues/6391).
+- Fixed a bug where the state of installing, updating, or uninstalling multiple extensions at once was lost when leaving the Marketplace screen.
+- Fixed issue where compose version in the about page would only get updated from v2 to v1 after restart.
+- Fixed an issue where user cannot not see log view because their underlying hardware didn't support WebGL2 rendering [docker/for-win#12825](https://github.com/docker/for-win/issues/12825).
+- Fixed a bug in user survey nagging too frequently.
+- Added setting under Settings -> General to override the OS's light mode / dark mode. Addresses (docker/for-win#12747)[https://github.com/docker/for-win/issues/12747].
```

Comment:
And this one is in the New section too.

Reply:
Looking at this, it's present in the new section as well, just worded differently:
```
Custom theme settings for Docker Desktop allow you to specify dark or light mode for Docker Desktop independent of your device settings.
```

Is it worth just adding the `for-win` link from this item on there?

Reply:
Yes, add the for-win link in the New section and delete this one.